### PR TITLE
Minor fixes after following readme

### PR DESCRIPTION
**Pull request by bstarling**
Add upgrade pip step
Correct typo in `get_pg_db_from_env`

### CODE REVIEW

File: docker/conda/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ RUN /opt/conda/bin/conda install jupyter -y --quiet && mkdir /opt/notebooks
 
 COPY ./source/python /python
 WORKDIR /python
+RUN pip install --upgrade pip
```
Comment (WanderingStar) [inline docker/conda/Dockerfile]: This step should come before the COPY, so the resulting layer can be cached.

**Review by WanderingStar — CHANGES_REQUESTED** [review]
Looks good, thanks. I'm surprised that a fresh install of the latest anaconda doesn't include the latest pip.

File: docker/conda/Dockerfile (hunk 1 above)
Comment (WanderingStar) [reply docker/conda/Dockerfile]: (Happy to explain further if my reasoning is not clear)

**Comment by bstarling** [comment]
Updated. Makes sense, thanks. I'm still very new to docker workflow.

**Review by WanderingStar — APPROVED** [review]

**Pull request closed without merging**